crates/ark/src/modules/positron/methods.R

```diff
@@ -26,38 +26,64 @@ lockEnvironment(ark_methods_table, TRUE)
 
 ark_methods_allowed_packages <- c("torch", "reticulate")
 
-#' Register the methods with the Positron runtime
-#'
-#' @param generic Generic function name as a character to register
-#' @param class Class name as a character
-#' @param method A method to be registered. Should be a call object.
-#' @export
-.ark.register_method <- function(generic, class, method) {
-    # Check if the caller is an allowed package
+# check if the calling package is allowed to touch the methods table
```

Comment:
There was just so much duplication between the register/unregister that I think these helpers might be nice?